universal-application-tool-0.0.1/app/models/Question.java

```diff
@@ -205,7 +205,9 @@ private void setQuestionOptions(QuestionDefinitionBuilder builder)
                 legacyQuestionOptions.get(firstKey).stream(),
                 (optionText, i) ->
                     QuestionOption.create(
-                        Long.valueOf(i), LocalizedStrings.of(firstKey, optionText)))
+                        Long.valueOf(i),
+                        Long.valueOf(i),
```

Comment:
This does not cover that case - but it doesn't need to, because that case is covered on line 195 - and also this code is obsolete, there are no longer any legacy questions anywhere as far as I can tell.